CafeTime/CafeTime/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,17 @@
+
+
+// login view
```

Comment:
it's optional, but I prefer to have kets with dots, like "remember.user.label"
and better to make it flow related, so "login.remember.user.label" will be perfect